src/test/java/spoon/getExecutableDeclarationBugTest/testClasses/ChangePasswordValidator.java

```diff
@@ -0,0 +1,43 @@
+package pt.ist.socialsoftware.edition.ldod.validator;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+import pt.ist.socialsoftware.edition.ldod.domain.LdoD;
+import pt.ist.socialsoftware.edition.ldod.domain.LdoDUser;
+import pt.ist.socialsoftware.edition.ldod.forms.ChangePasswordForm;
```

Comment:
Same as the above. Try to re-create the test case without any extra dependencies on external projects or files.